Src/Newtonsoft.Json/Bson/BsonWriter.cs

```diff
@@ -27,7 +27,7 @@
 using System.Collections;
 using System.Collections.Generic;
 using System.IO;
-#if !(NET20 || NET35 || PORTABLE40 || PORTABLE) || NETSTANDARD1_1
+#if HAVE_NUMERICS
```

Comment:
The only thing used from System.Numerics is BigInteger so this could be combined with HAVE_BIG_INTEGER